changes/2898.changed

```diff
@@ -0,0 +1 @@
+Disabled sorting on Computed Field column.
```

Comment:
IMO this should be a `.fixed` entry, since sorting didn't work beforehand anyway.